sui/README.md

```diff
@@ -147,6 +147,16 @@ The syntax is `node sui/gas-service.js payGas --amount <amount> <destinationChai
 node sui/gas-service.js payGas --amount 0.1 ethereum 0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05 0xba76c6980428A0b10CFC5d8ccb61949677A61233 0x1234
 ```
 
+Collect gas:
+
+Conditions:
+- The `GasCollectorCap` must be stored in the operator contract.
```

Comment:
```suggestion
- The `GasCollectorCap` must be stored in the gas service object config.
```
This should be saved in the config under the GasService objects (if it's not already being saved), and operators script should read from it instead of needing a copy.

Reply:
Sorry, I meant the `store_cap` in Operators contract must be called first.